### DISCUSSION TITLE
is `list-disc` not working, or what am I doing wrong?

### DISCUSSION THREAD

**Discussion by acoyfellow** [tailwindlabs/tailwindcss#11276] [Help]
What am I misunderstanding here.. I can't get `list-disc` to work?

```
<ul class="list-disc">
  <li>foo</li>
</ul>
```

Reproduction: https://play.tailwindcss.com/nzvnJV9wT0

**Comment by wongjn** [comment]
It may be that you cannot visually see the list marker. [Here is a slight modification of the layout to make the list marker visible](https://play.tailwindcss.com/OVI07UtdWY).

**Reply by ahmetcanisik** [reply]
> It may be that you cannot visually see the list marker. [Here is a slight modification of the layout to make the list marker visible](https://play.tailwindcss.com/OVI07UtdWY).

You can also wrap your ul in a p tag or try what @wongjn said.

**Comment by mybearworld** [comment]
This is an annoyance with CSS where by default, the list marker actually appears outside of the element. You can use `list-inside` to fix this: https://play.tailwindcss.com/9GvAY49hfW

**Reply by scherroman** [reply]
`list-inside` looks ugly with longer list items though, it doesn't keep the next lines indented properly :/

`list-disc` with an offset of `ml-4`:
<img width="710" alt="Screenshot 2024-07-31 at 5 41 30 PM" src="https://github.com/user-attachments/assets/6e59df8a-df38-42a4-a9fd-cf7b8c21d414">


`list-disc` with `list-inside`:
<img width="710" alt="Screenshot 2024-07-31 at 5 41 40 PM" src="https://github.com/user-attachments/assets/1eb1c939-a9a7-4a3d-9564-24e9da04a865">

**Comment by sonik-auctane** [comment]
This is a weird behaviour I encountered as well. I hope this is helpful to you:
```
<ul class="list-disc list-inside">
  <li class="list-item">item WITH disc. ✅</li>
  <li>item WITHOUT disc. ❌</li>
</ul>
```

**Reply by zolboo-b** [reply]
made my day

**Reply by Langlw1992** [reply]
worked for me, thanks a lot

**Comment by maurer2** [comment]
I believe you also loose the list marker if you change a list item's default display value from `display: list-item` to any other value like, e.g. `block`, `flex`, `grid` etc.

**Reply by Varshni11** [reply]
Do you know any workaround or solution for this?

**Reply by wongjn** [reply]
You could consider:
- Faking the list marker with a pseudo-element.
- Wrapping the content of the `<li>` element in another element and applying the alternative `display` value to that element.

**Reply by Varshni11** [reply]
Thanks for your suggestions!For me, your first suggestion worked, but the second one created weird results in UI, like all the content inside the div under li appeared in the next line after the bullet.

**Reply by wongjn** [reply]
For the second suggestion, you'd probably need to apply more CSS around it to get it to display as you'd like.

**Comment by mohammadazeemwani** [comment]
Greetings. 

with `list-disc`, there needs to be some padding left < default it is `0` ig.
So this should fix the issue:

```html
<ul class="list-disc pl-7">
  <li>Item First</li>
  <li>Item Second</li>
</ul>
```

**Reply by CheeryCodes** [reply]
Thank you
